tests.py

```diff
@@ -2,40 +2,51 @@
 import os
 import requests
 import subprocess
+import sys
 import time
 import unittest
 
 
 class CommandlineTest(unittest.TestCase):
+
     def setUp(self):
-        command = "docker port container-{STAMP}{SUFFIX} | perl -pne 's/.*://'".format(**os.environ)
-        os.environ['PORT'] = subprocess.check_output(command, shell=True).strip().decode('utf-8')
-        self.url = 'http://localhost:{PORT}/api/v1/tilesets/'.format(
-            **os.environ)
+        command = "docker port container-{STAMP}{SUFFIX} | " \
+            "perl -pne 's/.*://'".format(**os.environ)
+        os.environ['PORT'] = subprocess.check_output(
+            command, shell=True).strip().decode('utf-8')
+        self.base_url = "http://localhost:{PORT}/".format(**os.environ)
+        self.tilesets_url = '{}api/v1/tilesets/'.format(self.base_url)
         while True:
-            if 0 == subprocess.call('curl --fail --silent '+self.url+' > /dev/null', shell=True):
+            if 0 == subprocess.call(
+                'curl --fail --silent ' + self.tilesets_url + ' > /dev/null',
+                shell=True
+            ):
```

Comment:
I'm embarrassed to have written all of this. See https://github.com/refinery-platform/refinery-developer-vis-tool/blob/master/test.py#L26-L42